src/product/index.ts

```diff
@@ -20,7 +20,7 @@ class Product {
         variants: []
       });
     },
-    onClick: () => this.flux.details(this.state.data),
+    onClick: () => this.flux.detailsWithRouting(this.select(Selectors.productWithId, this.state.data.id)),
```

Comment:
I'm not sure if there's any way to undo a product transformation but basically I just did this to get the whole product in the store.

Reply:
Why do you need the non-transformed version?

Reply:
Because gb-product expects an untransformed product so it can run productTransformer on it. If given an already transformed product it gives this: https://i.imgur.com/DCKnWJs.png. I'm wary of allowing transformed products in the store data section anyway since they have less data than the full product and it complicates logic as we may run into situations where we need some data from a product but we just don't have it.

Reply:
Okay so I'm guessing moving the product transformer into the store will solve this problem?

Reply:
No, as it's still in core, it didn't actually get moved into flux, but I think it might be a good idea in the product transformers to get the whole product if any necessary data is missing, it simplifies the logic here. Should that be separate ticket?